### PR TITLE
feat: ens profile pic

### PR DESCRIPTION
**Pull request by defispartan**
Display ENS avatar in the top right AddressInfo box, and slightly modify existing style for the address box and dropdown.

Design: https://www.figma.com/file/0ClevOXCXZioRVpogcNMFh/ENS-Profile-Avatar?node-id=229%3A3580

Also move the call to the ENS hook from address panel to staticProvider to avoid reloading the name/avatar on network change.

### CODE REVIEW

File: src/components/menu/AddressInfo/index.tsx [hunk 1]
```diff
@@ -1,22 +1,31 @@
...
+  gradient,
+  rgba,
+  textCenterEllipsis,
+  useThemeContext,
+} from '@aave/aave-ui-kit';
 
 import { useUserWalletDataContext } from '../../../libs/web3-data-provider';
 import { useMenuContext } from '../../../libs/menu';
+import { getNetworkConfig } from '../../../helpers/config/markets-and-network-config';
+import { useStaticPoolDataContext } from '../../../libs/pool-data-provider';
```
Comment (foodaka) [inline src/components/menu/AddressInfo/index.tsx:17]: surprised we arent using some root import file tool

Comment (defispartan) [reply src/components/menu/AddressInfo/index.tsx:17]: Yeah, we should probably setup absolute imports and use them everywhere

Comment (defispartan) [reply src/components/menu/AddressInfo/index.tsx:17]: Created #207 for this

File: src/libs/hooks/use-get-ens.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+      console.error('ENS name lookup error', error);
+    }
+  };
+
+  const getAvatar = async (name: string) => {
+    try {
+      const labelHash = utils.keccak256(utils.toUtf8Bytes(name?.replace('.eth', '')));
+      const result: { background_image: string } = await (
+        await fetch(
+          `https://metadata.ens.domains/mainnet/0x57f1887a8BF19b14fC0dF6Fd9B2acc9Af147eA85/${labelHash}/`
```
Comment (foodaka) [inline src/libs/hooks/use-get-ens.tsx:32]: which address is this?

Comment (defispartan) [reply src/libs/hooks/use-get-ens.tsx:32]: It's the ENS base registry contract

**Pull request merged**